skll/config.py

```diff
@@ -582,6 +581,23 @@ def _parse_config_file(config_path, log_level=logging.INFO):
     # using the defaults?
     do_grid_search = config.getboolean("Tuning", "grid_search")
 
+    # Check if `param_grids` is specified, but `grid_search` is False
+    if param_grid_list and not do_grid_search:
+        logger.warning('"param_grids" was specified despite the fact that '
```

Comment:
I'd rephrase this to state simply that `Since "grid_search" is set to False, the specified "param_grids" will be ignored.`.